### PR TITLE
Add note to 0.16.0 release changelog about embedded JDK

### PR DESCRIPTION
**Pull request by jin**

### CODE REVIEW

**Comment by jin** [comment]
cc @cushon

File: _posts/2018-07-31-bazel-0.16.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Here are the main changes of Bazel 0.16:
 *   We have added the [`--materialize_param_files`](https://docs.bazel.build/versions/master/command-line-reference.html#flag--materialize_param_files) flag to write parameter files even when actions are executed remotely.
 *   `--[no]allow_undefined_configs` no longer exists, passing undefined configs will result in an error.
 *   We have added the attribute `repo_mapping` for repository rules that enables dependencies to be shadowed. Use `--experimental_enable_repo_mapping` to try it out. More details [here](https://docs.bazel.build/versions/master/external.html#shadowing-dependencies)
+* The embedded JDK has been upgraded to JDK 9, and is now used as both the default startup `--host_javabase` and build `--host_javabase`.
```
Comment (cushon) [inline _posts/2018-07-31-bazel-0.16.md]: Can you add links?

https://docs.bazel.build/versions/master/user-manual.html#flag--host_javabase
https://docs.bazel.build/versions/master/user-manual.html#startup_flag--host_javabase

Comment (jin) [reply _posts/2018-07-31-bazel-0.16.md]: done

**Comment by jin** [comment]
Thanks @cushon

**Pull request merged**